swatch-test-framework/src/main/java/com/redhat/swatch/component/tests/core/BaseServiceTest.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Red Hat, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ * Red Hat trademarks are not licensed under GPLv3. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+package com.redhat.swatch.component.tests.core;
+
+import com.redhat.swatch.component.tests.api.ComponentTest;
+import com.redhat.swatch.component.tests.api.KafkaBridgeService;
+import java.util.Collections;
+import java.util.Set;
+import org.junit.jupiter.api.AfterEach;
+
+@ComponentTest
+public abstract class BaseServiceTest {
```

Comment:
why do you need this extra parent level? All the tests in swatch-producer-azure/ct are to be used for azure only, therefore BaseAzureTest should be enough (and move the cleanupAfterEachTest method there.

Reply:
Other services won't see this BaseServiceTest because it's created in swatch-producer-azure/ct.